src/org/jitsi/impl/neomedia/rtp/MediaStreamTrackDesc.java

```diff
@@ -29,23 +31,47 @@
 public class MediaStreamTrackDesc
 {
     /**
+     * The system property name that holds the minimum time (in millis) that is
+     * required for the media engine to generate a new key frame.
```

Comment:
also, i assume the 'wait' here is from a previous keyframe request?  or is this a delay from when the user joins/stream is created?